wire-ios-data-model/Source/Model/Conversation/ZMConversation+SecurityLevel.swift

```diff
@@ -676,6 +677,23 @@ extension ZMConversation {
         return self.localParticipants.first { $0.clients.count == 0 } == nil
     }
 
+    fileprivate var hasMoreClientsThanSelfClient: Bool {
+        guard
+            let context = managedObjectContext,
+            let selfClient = ZMUser.selfUser(in: context).selfClient()
+        else {
+            return false
+        }
+
+        let clients = localParticipants.flatMap(\.clients)
+
+        if clients.contains(selfClient) && clients.count == 1 {
+            return false
+        }
+
+        return true
```

Comment:
```suggestion
        return !(clients.contains(selfClient) && clients.count == 1)
```
shorter, though might be less easier to read, your call